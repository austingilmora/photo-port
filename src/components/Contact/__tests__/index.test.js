import React from "react";
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ContactForm from "..";

afterEach(cleanup)

describe('ContactForm', () => {
    it('renders', () => {
        render(<ContactForm />)
    });

    it('matches snapshot', () => {
        const { asFragment } = render(<ContactForm />); 
        expect(asFragment).toMatchSnapshot();
    });
});

describe('h1 is right', () => {
    it('has contact me in h1', () => {
        const { getByTestId } =render(<ContactForm />);

        expect(getByTestId('contact')).toHaveTextContent('Contact Me')
    });
});

describe('contactButton says submit', () => {
    it('says submit', () => {
        const { getByTestId } =render(<ContactForm />);

        expect(getByTestId('contactButton')).toHaveTextContent('Submit')
    })
})