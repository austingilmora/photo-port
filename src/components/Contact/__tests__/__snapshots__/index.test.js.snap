// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactForm matches snapshot 1`] = `[Function]`;
